docs/manual/wcur2.py

```diff
@@ -1,7 +1,7 @@
-    def get_pref_col(self, (maxcol,)):
-        return self.cursor_x
-
-    def move_cursor_to_coords(self, (maxcol,), col, row):
-        assert row == 0
-        self.cursor_x = col
-        return True
+# def get_pref_col(self, (maxcol,)):
+#         return self.cursor_x
+#
+# def move_cursor_to_coords(self, (maxcol,), col, row):
+#     assert row == 0
+#     self.cursor_x = col
+#     return True
```

Comment:
commenting out manual examples?

Reply:
Problem, that it's part of class copy-paste without class declaration -> this code causes parsing error